src/Test/L0/SecretMaskerL0.cs

```diff
@@ -4,13 +4,14 @@ namespace Microsoft.VisualStudio.Services.Agent.Tests
 {
     public sealed class SecretMaskerL0
     {
+        // TODO: Fix these test names.
         [Fact]
         [Trait("Level", "L0")]
         [Trait("Category", "Common")]
         public void SecretMasker_MaskSecrets_EmptyInput()
         {
             var secretMasker = new SecretMasker();
-            secretMasker.Add(new ValueSecret("abcd"));
```

Comment:
The Add method was only called from test code. I removed the method from the interface and switch the test code to use the production interface.